Rakefile

```diff
@@ -10,12 +9,33 @@ end
 
 task :default => :test
 
+desc "Bundle, tag, and commit after you update the version"
+task :version => [:bundle_all] do
```

Comment:
ideally hook into bump to do that, not sure if that's easy ...

Reply:
... I guess bump needs multi-gemfile support :/
... maybe append the bump tasks to bundle all and then `git commit --amend --no-edit` ?
```
task "bump:patch" do
  Rake::Task["bundle_all"].invoke
  sh "git commit -a --amend --no-edit"
end
```